usecase/model/ibc/ibc.go

```diff
@@ -45,8 +47,8 @@ type Packet struct {
 }
 
 type FungibleTokenPacketData struct {
-	Sender   string `mapstructure:"sender" json:"sender"`
-	Receiver string `mapstructure:"receiver" json:"receiver"`
-	Denom    string `mapstructure:"denom" json:"denom"`
-	Amount   uint64 `mapstructure:"amount" json:"amount,string"`
+	Sender   string      `mapstructure:"sender" json:"sender"`
+	Receiver string      `mapstructure:"receiver" json:"receiver"`
+	Denom    string      `mapstructure:"denom" json:"denom"`
+	Amount   json.Uint64 `mapstructure:"amount" json:"amount"`
```

Comment:
The wire format actually contains `amount` as  `string`. I hope this'll work with `string`?

Reply:
I think so @devashishdxt, tested a bit on my local and its green. `json.Uint64` should aims to deal with both `uint64` and `string` format